docs/audio.rst

```diff
@@ -14,18 +17,37 @@ which means that it can reproduce frequencies up to 3.9kHz.
 Functions
 =========
 
-.. py:function:: play(source, wait=True, pin=pin0, return_pin=None)
+.. py:function:: play(source, wait=True, pin=pin_speaker, return_pin=None)
 
     Play the source to completion.
 
-    ``source`` is an iterable, each element of which must be an ``AudioFrame``.
+    ``source`` is an iterable, each element of which must be an ``AudioFrame``
```

Comment:
Does L25 cover this or maybe it should be `can be an ``AudioFrame`` or built-in ``Sound``.

Reply:
I'd probably open with all the different types the argument can take, and then describe them. We might have to expand this in the future as well, so that format also helps that.

Reply:
I've updated this, but not sure if we want something more structured, maybe like this
```
:Param source: ``Sound`` or iterable of ``AudioFrame``

* ``Sound`` - The ``microbit`` module contains a list of built in sounds.
```
etc

We could also use the `:Param` format for the microphone docs

Reply:
Does that format the HTML as well? Do you have a screenshot?
If it does it sounds like a good feature to start using.

Reply:
Not sure this is formatted correctly, but it doesn't look as good as I hoped

![image](https://user-images.githubusercontent.com/31242877/100432722-eb3d8200-3091-11eb-922b-2684a334c818.png)

Reply:
Expected it to be similar to https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html

![image](https://user-images.githubusercontent.com/31242877/100434210-0c06d700-3094-11eb-9e55-1e5b19e7f822.png)

Reply:
Great, using param is good, specially if we move to pyi files in the future.